include/fmt/core.h

```diff
@@ -1613,6 +1633,180 @@ inline void print(const S& format_str, Args&&... args) {
       internal::make_args_checked<Args...>(format_str, args...));
 #endif
 }
+
+namespace internal {
+
+template <typename T, typename Char> struct is_string_view : std::false_type {};
+
+template <typename Char>
+struct is_string_view<basic_string_view<Char>, Char> : std::true_type {};
+
+template <typename Char>
+struct is_string_view<std_string_view<Char>, Char> : std::true_type {};
+
+template <typename T> struct is_ref_wrapper : std::false_type {};
+
+template <typename T>
+struct is_ref_wrapper<std::reference_wrapper<T>> : std::true_type {};
+
+template <typename T, typename Context> struct need_dyn_copy {
+  static constexpr internal::type mapped_type =
+      mapped_type_constant<T, Context>::value;
+  static_assert(mapped_type != internal::type::named_arg_type,
+                "Bug indicator. Named arguments must be processed separately");
+
+  using type = std::integral_constant<
+      bool, !(is_ref_wrapper<T>::value ||
+              is_string_view<T, typename Context::char_type>::value ||
+              (mapped_type != internal::type::cstring_type &&
+               mapped_type != internal::type::custom_type &&
+               mapped_type != internal::type::string_type))>;
+};
+
+class dyn_arg_storage {
+  // Workaround clang's -Wweak-vtables. For templates (unlike regular classes
+  // doesn't complain about inability to deduce translation unit to place vtable
```

Comment:
I had problems parsing the above. How about rephrasing to something like:

```
  // Workaround clang's -Wweak-vtables for templates (unlike regular classes)
  // don't complain about inability to deduce translation unit to place vtable.
```

Reply:
Will fix.. Not exactly like you wrote but somehow...